dbarashinvd/sonic-buildimage
# fix hw_reset low polarity (reverse values)

#### Why I did it
fix low polarity wrong value for hw_reset deassert as part of power_on_module state function

##### Work item tracking
- Microsoft ADO **(number only)**:

#### How I did it
send 1 to hw_reset sysfs instead of existing 0 value sent to deassert hw_reset after power_on when needed

#### How to verify it
run the code in manual test of plugged out cable when the switch it booted.
then plug in the cable after module detection is done and dynamic detection takes place listening to sysfs changes via poller
for optic CMIS cable CMIS it will not be powered on thus will go into power_on_module function with the above change here.
at the end of module detection for this new cable inserted, EEPROM should be available for reading (currently with 0 sent to hw_reset EEPROM is not available at the end)


#### Which release branch to backport (provide reason below if selected)




#### Tested branch (Please provide the tested image version)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
